app/helpers/automatic/daily_room_pool.py

```
@@ -0,0 +1,505 @@
+
+            # Get tokens with the shorter, session-specific expiry.
+            token_params = self._get_token_params(use_session_limit=True)
+            user_token = await self.daily_rest_helper.get_token(
+                room.url,
+                expiry_time=self.max_session_limit,
+                eject_at_token_exp=True,
+                owner=False,
+                params=token_params,
+            )
+            if not user_token:
+                raise RuntimeError("Failed to get user token")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)